rules/windows/process_creation/proc_creation_win_msdt_diagcab.yml

```diff
@@ -15,8 +18,8 @@ detection:
         - OriginalFileName: 'msdt.exe'
     selection_cmd:
         CommandLine|contains|all:
-            - '/cab'
-            - '.diagcab'
+            - ' /cab'
+            - ' -cab'
     condition: all of selection_*
```

Comment:
this cannot be `|contains|all` after the proposed change --> will be false almost always

Reply:
i changed it